src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import classes from './Sidebar.module.scss';
+import { Link, useLocation } from 'react-router-dom';
+import {
+  HomeIcon,
+  UsersIcon,
+  UserPlusIcon,
+  ChatBubbleLeftRightIcon,
+  DocumentPlusIcon,
+  UserIcon,
+  ArrowRightStartOnRectangleIcon
+} from '@heroicons/react/24/solid';
+
+const Sidebar: React.FC = () => {
+  const location = useLocation();
+  return (
+    <div className={`${classes['c-sidebar']}`}>
```

Comment:
This should be a bit more modular, the way you set everything up it is really hustle to add new sidebar elements. My suggestion is to receive list of objects here and then render them